draft-ietf-tls-tls13.md

```diff
@@ -4177,6 +4180,13 @@ Cryptographic details:
   by the server are acceptable (see
   {{diffie-hellman-key-exchange-with-authentication}})?
 
+- When making ECDSA signatures, consider countermeasures against failure
+  to properly choose the per-message secret "k" value.  A RECOMMENDED
+  countermeasure is described in {{RFC6979}}.
+  The impact of a failure here is key material disclosure, and in TLS the
+  keys involved are typically high value: long-term, externally certified
+  and the root of endpoint authentication.
```

Comment:
These last 3 lines should probably be cut, as this is a bit long and overly redundant. Also, the other bullet points are in the form of a question, and this isn't (just a nitpick, though).